File: JavaScript/exercicio-02/solucao.js
// Bibliotecas
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {
  
  function dizerOla() {

    const nomePessoa = prompt("Qual é o seu nome ? ");
    console.log(`Olá ${nomePessoa}, é um prazer te conhecer !`);
    
  }
  
  dizerOla();
}


main();
readline.question('Pressione Enter para sair ... ');

File: C#/exercicio02/solucao.cs
using System;

public class Program
{
    public static void Main()
    {
        DizerOla();
    }

    static void DizerOla()
    {
        Console.WriteLine("Digite o seu nome: ");
        string nome = Console.ReadLine(); 

        Console.WriteLine($"Olá {nome}, seja bem vindo!");

        Console.WriteLine("Pressione Enter para sair...");
        Console.ReadLine();
    }
}
